src/main/resources/META-INF/hudson.remoting.ClassFilter

```diff
@@ -0,0 +1 @@
+org.jenkins.plugins.lockableresources.LockObject
```

Comment:
Delete this. It should not be necessary to whitelist a class defined in this plugin.